src/foam/nanos/iban/ValidationIBAN.js

```diff
@@ -506,8 +510,8 @@ Columns: validation format, parsing format, example`,
             case 't': // account type
               accountType.append(next);
               break;
-            case 'x': // add to previous
-              previous.append(next);
+            case 'x': // national check digits
+              // previous.append(next);
```

Comment:
Why are you removing this?

Reply:
`x` is just a check digit in IBAN eg., https://bank.codes/iban/structure/portugal/ might not be contributed to the previous parsed field.

Still need to confirm with @nanoNeel and Nadir on Monday.

Reply:
It's just a check-digit, but it's still part of the #.

Reply:
Zeldi is helping with the bank account number format for all the countries we are supporting, then should be able to confirm whether check digit is part of account number.

Reply:
It is part of the previous number sequence. Removing this is why you needed to change the x to c on line 47.